### ISSUE TITLE
Provide a way to mark a piano introduction

### ISSUE THREAD

**Issue by samuelbradshaw** [w3c/musicxml#275]
This is an issue that applies mainly to hymnals. In many hymnals, brackets are used to mark an introduction of the hymn, played by the piano or organ before the singing begins. This isn't universal, but is relatively common.*

It would be helpful if there were a standard way to mark this up in MusicXML – especially if it could be done semantically, in a way that audio playback applications would know to play that part first before playing the first verse.

![hymn-intro-brackets](https://user-images.githubusercontent.com/7323052/54478498-67b60180-47d8-11e9-989b-e9151ea09fbe.png)

*The example above is from Hymns of The Church of Jesus Christ of Latter-day Saints (1985). A few other hymnals that use the same symbols for marking an introduction: Celebrating Grace Hymnal (2010), Hymns of Grace (2015), The Baptist Hymnal (1991), The Christian Life Hymnal (2006).

**Comment by mdgood** [comment]
Those symbols are available in MusicXML in the `<principal-voice type="plain"/>` element. I believe you could get the representation you want by using those directions, combined with a sound element with the appropriate segno and dalsegno attributes.

**Comment by Kreshniku21** [comment]
@mdgood where would one find more information about how principal-voice element functions and how to edit it?  Also what if the Piano introduction appears only at the end or the middle of a written hymn?  If that is the case is there a way to tag the XML so that it would begin playback on a measure other than the first one?  You are right however that segno and dalsegno attributes work for the example posted above.

**Comment by samuelbradshaw** [comment]
Thanks! I was able to get brackets to show in the score by putting
```
      <direction placement="above">
        <direction-type>
          <principal-voice type="start" symbol="plain"/>
        </direction-type>
      </direction>
```
before the `<note>` where I wanted the introduction to start, and
```
      <direction placement="above">
        <direction-type>
          <principal-voice type="stop" symbol="plain" />
        </direction-type>
      </direction>
```
where I wanted the introduction to end.

Finding a MusicXML viewer that shows the brackets hasn't been as fruitful. Soundslice's online viewer doesn't seem to render the brackets. Importing into MuseScore and a Finale viewer didn't show them either. SeeScore's demo iPad app does render them, which was the only way I was able to confirm that it worked (thanks, SeeScore!). So, support is limited.
